src/homekit_decl.c

```diff
@@ -51,7 +53,7 @@ homekit_characteristic_t obstruction_detected = HOMEKIT_CHARACTERISTIC_(
 homekit_accessory_t *accessories[] = {
     HOMEKIT_ACCESSORY(.id=1, .category=homekit_accessory_category_garage, .services=(homekit_service_t*[]){
             HOMEKIT_SERVICE(ACCESSORY_INFORMATION, .characteristics=(homekit_characteristic_t*[]){
-                    HOMEKIT_CHARACTERISTIC(NAME, "Garage Door"),
+                    HOMEKIT_CHARACTERISTIC(NAME, device_name),
                     HOMEKIT_CHARACTERISTIC(MANUFACTURER, "ratCloud llc"),
                     HOMEKIT_CHARACTERISTIC(SERIAL_NUMBER, "123456"),
```

Comment:
Should this also be set to the chip ID?

Reply:
It should already be, `device_name` is getting the chip ID from `homekit.cpp`

Reply:
sorry, I was referring to the serial number

Reply:
Ah gotcha. Added chipId to `SERIAL_NUMBER`.